src/factories/nodeFactory.ts

```diff
@@ -2,14 +2,15 @@ import { INode, ICoordinate } from '../models';
 import NodeType from '../enums/nodeType';
 
 class NodeFactory {
-    public static createNode = (node: any): INode => {
+    public static createNode = (routePathId: string, node: any): INode => {
         const coordinateList = JSON.parse(node.solmuByLnkalkusolmu.geojson);
         const coordinate : ICoordinate = {
             lon: coordinateList.coordinates[0],
             lat: coordinateList.coordinates[1],
         };
 
         return <INode>{
+            routePathId,
```

Comment:
Should this be renamed so that its clear that its generated id and not from jore?

I would rename as:
internalRoutePathId
or
generatedRoutePathId